packages/frontend/src/App.tsx

```diff
@@ -0,0 +1,26 @@
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import ErrorPage from '~/shared/components/ErrorPage';
+import LoginPage from '~/modules/auth/LoginPage';
+import { Box } from '@mui/material';
+
+const App = () => {
+  const router = createBrowserRouter([
```

Comment:
What do you think about putting routes into their own file (e.g. `routes.tsx`)? 

To your fourth question, I think more routes and maybe some examples of how to use the actions/loaders would definitely be beneficial if there’s time for it. Any thoughts on page transitions with this setup using Framer?

Reply:
I think importing the list of routes makes sense, I'll refactor.

A wrinkle on the actions/loaders: it seems like this pattern in react-router v6 clashes with Apollo's `useQuery` hook. (relevant thread [here](https://community.apollographql.com/t/how-to-load-data-with-react-router-dom-v6-and-handling-errorpage/5347/3)). If I'm understanding correctly, the actions/loaders approach removes our ability to get cache updates as new values are updated, unless we jump through some (in my opinion rather convoluted) hoops.

Until there's a clear happy path here, I lean toward continuing to just do data fetching with the `useQuery` hook.

I'll take a closer look at the page transitions stuff you sent me today, but will probably create a separate issue for implementing that.